packages/lpa-questionnaire-web-app/src/views/layouts/multi-file-upload.njk

```diff
@@ -114,10 +116,34 @@
 window.MOJFrontend.initAll();
 
 if (typeof MOJFrontend.MultiFileUpload !== 'undefined') {
+  var temporaryDocInput = document.getElementById('tempDocs');
+
+  function onFileUpload(context, file, response) {
+    var upload = {
+      id: response.file.id,
+      name: response.file.originalname,
+      error: response.error ? response.error.message : undefined,
+    };
+
+    temporaryDocInput.value = temporaryDocInput.value + JSON.stringify(upload);
+  };
+
+  function onFileDelete(context, response) {
+    var rawUploads = temporaryDocInput.value.match(/{.*?}/g) || [];
```

Comment:
same as above - worth a sanity check on `temporaryDocInput.value` being defined and a string?

Reply:
Likewise, separate event for unhappy response